13.json

```diff
@@ -755,7 +755,7 @@
                     ]
                   },
                   "direction": {
-                    "description": "The wind direction in degrees. Use this <a href=\"https://github.com/slopjong/OpenSpaceLint/issues/80\" target=\"_blank_\">mapping</a> to convert the degrees into a string.",
+                    "description": "The wind direction in degrees. To convert from abbreviation to degrees you can use <a href=\"https://en.wikipedia.org/wiki/Points_of_the_compass#32_compass_points\">this table.</a>",
```

Comment:
Is this conversion even relevant? Degrees seem straigthforward to me.

Reply:
Don't know, pretty sure you will have an idea if you're having a wind sensor anyways, but didn't want to drop the information how to do that. 
Fyi, right now no one provides a wind sensor right now.

Reply:
I've never seen a wind sensor that provides the wind direction as a string, it's always an angle. I'd remove the link to WP, it's probably more confusing than helpful :)